app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -378,25 +432,28 @@ Parameter ¹                          | Description
 `logout_revoke`                      | Revoke tokens from IdP on logout by calling `revocation_endpoint`.
 `logout_revoke_access_token`         | If `logout_revoke` is turned on, this controls whether or not `access_token` is revoked on logout.
 `logout_revoke_refresh_token`        | If `logout_revoke` is turned on, this controls whether or not `refresh_token` is revoked on logout.
-`revocation_endpoint`                | If `revocation_endpoint` is not specified in discovery (as it is not standardized by OpenID Connect), you can specify it manually.
-`end_session_endpoint`               | If `end_session_endpoint` is not specified in discovery, you can specify it manually (e.g. you can use your own non-OpenID Connect logout endpoint).
-`token_exchange_endpoint`            | Token exchange endpoint, if you want to exchange the access token to a new one before proxying to upstream service.
+`credential_claim`                   | Name of the claim that is used to set arbitrary credential (can be used with e.g. rate-limiting).
```

Comment:
```suggestion
`credential_claim`                   | The name of the claim that is used to set an arbitrary credential, e.g., it can be used with rate-limiting).
```